### PR TITLE
Bug fix/adapt socketcan connectors to be agnostic

### PR DESCRIPTION
**Pull request by BKaDamien**
CCSocketCan is now agnostic

### CODE REVIEW

File: src/pykiso/lib/connectors/cc_socket_can/cc_socket_can.py [hunk 1]
```diff
@@ -209,12 +208,12 @@ def _cc_receive(
                         frame_id, payload, timestamp
                     )
                 )
-                return payload, frame_id
+                return {"msg": payload, "remote_id": frame_id}
```
Comment (Pog3k) [inline src/pykiso/lib/connectors/cc_socket_can/cc_socket_can.py:211]: adapt function type hint

File: docs/whats_new/version_ongoing.rst [hunk 2]
```diff
@@ -43,3 +43,7 @@ adapt the record auxiliary to fit with the brand new double threaded auxiliary i
 Acroname Auxiliary adaption
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 adapt the acroname auxiliary to fit with the brand new double threaded auxiliary interface
+
+Agnostic CCSocketCan
```
Comment (Pog3k) [inline docs/whats_new/version_ongoing.rst:47]: The  "whats new " should actually only contain new features and not fixes. So the most interesting part for users.

You should squash everything down to one single commit message, so the changes will then appear in the Changelog.
fix: ...

**Review by Pog3k — CHANGES_REQUESTED** [review]
Squash commits.
Remove whats new
Update Type hint

File: tests/test_cc_socket_can.py [hunk 3]
```diff
@@ -193,27 +193,27 @@ def test_cc_close(logging_requested, mock_can_bus, mocker):
...
     [
-        (b"\x10\x36", 0x0A, True),
-        (b"\x10\x36", None, True),
-        (b"\x10\x36", 10, True),
-        (b"", 10, True),
-        (message_with_tlv, 0x0A, False),
-        (message_with_no_tlv, 0x0A, False),
-        (message_with_no_tlv,),
-        (message_with_no_tlv, 36),
+        {"msg": b"\x10\x36", "remoet_id": 0x0A, "raw": True},
```
Comment (sebclrsn) [inline tests/test_cc_socket_can.py:196]:
```suggestion
        {"msg": b"\x10\x36", "remote_id": 0x0A, "raw": True},
```
Typo to be fixed in all parameters

**Review by sebclrsn — CHANGES_REQUESTED** [review]

**Comment by sebastianpfischer** [comment]
@dependabot squash

**Comment by sebastianpfischer** [comment]
Squash comes after (while merging)

**Review by sebclrsn — APPROVED** [review]

**Pull request merged**